tendrl/ceph_integration/ceph.py

```diff
@@ -380,9 +380,14 @@ def rbd_command(command_args, pool_name=None):
 
     p = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
                          stdin=open(os.devnull, "r"))
+    if p.poll() is None: # Force kill if process is still alive
```

Comment:
How much seconds before you consider the rbd cmd as not responding due to cluster in ERROR state?